rpc/src/rpc.rs

```diff
@@ -1853,15 +1969,25 @@ impl JsonRpcRequestProcessor {
         Ok(new_response(&bank, balance))
     }
 
-    pub fn get_token_supply(
+    pub async fn get_token_supply(
         &self,
         mint: &Pubkey,
         commitment: Option<CommitmentConfig>,
     ) -> Result<RpcResponse<UiTokenAmount>> {
         let bank = self.bank(commitment);
-        let mint_account = bank.get_account(mint).ok_or_else(|| {
-            Error::invalid_params("Invalid param: could not find account".to_string())
-        })?;
+        let mint_account = self
+            .runtime
+            .spawn_blocking({
+                let bank = Arc::clone(&bank);
+                let mint = Pubkey::clone(mint);
+                move || {
+                    bank.get_account(&mint).ok_or_else(|| {
```

Comment:
Token accounts are small, so not sure that we should send this to the bg pool?
Especially considering it's memecoin szn so this is probably going to be called
a lot? We must be careful to not put _too_ many things in the bg pool, or we
accidentally serialize frequent, fast requests behind chonky, slow ones we put
in the bg pool